doc/source/ray-air/examples/dreambooth_finetuning.rst

```diff
@@ -16,8 +16,8 @@ See the HuggingFace tutorial for useful explanations and suggestions on hyperpar
 
 **Compute requirements:**
 
-* Because of the large model sizes, you'll need a machine with at least 2 A10G GPUs.
-* Each training worker uses 2 GPUs, so you'll need ``N * 2`` total GPUs in your Ray cluster, if training with ``N`` distributed training workers. The example can leverage data-parallel training with more GPUs to speed up training time.
+* Because of the large model sizes, you'll need a machine with at least 1 A10G GPU.
```

Comment:
Quick question, why does this not need 2 GPUs anymore? If this is only applicable when LoRA is enabled I would be explicit about this.

Reply:
No applicable to non-lora case as well. We decided it this way, so the example can directly use `prepare_model` and not a lot of manual stuff. Less noise.